trite-osprey/src/App.css

```diff
@@ -0,0 +1,156 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 10s linear;
+  height: 80px;
+  filter: invert(100%);
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-intro {
+  font-size: large;
+  padding: 1em;
+}
+h2{
+  font-family: 'Permanent Marker', cursive;
+  font-size: 3em;
+
+}
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+.listclass {
+  display: flex;
+  align-items: center;
+  z-index: 1;
+}
+
+input[type = text]{
+  text-align: center;
+  padding: 2px 2px;
+  box-shadow: 0 1px #ddd;
+  width:384px;
+  outline: none;
+  display:block;
+  color:#222;
+  margin: 0 auto 20px;
+  margin-top: 20px;
+  font-family: 'Permanent Marker', cursive;
+  font-weight: lighter;
+}
+
+.center_section{
+  display: flex;
+  justify-content: space-around;
+
+}
+
+.container_left{
+  display:flex;
+  justify-content: space-around;
+}
+
+.container_right{
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  flex-direction: column;
+}
+
+.popout {
+  display:flex;
+  flex-direction: column;
+  z-index: 5;
+  width: 450px;
+  height: 400px;
+  border:6px solid #222;
+  background-color: #fff;
+  position: absolute;
+  top: 275px;
+  left: 320px;
+  font-family: 'Permanent Marker', cursive;
+  font-weight: lighter;
+}
+
+.invisible {
```

Comment:
this utility class is often called `.hidden` instead of `.invisible` because visibility is different from not being displayed